tasks/router/router_1.ts

```diff
@@ -9,7 +9,7 @@ import {
 import { task } from "hardhat/config";
 import { encodeFunctionData } from "viem";
```

Comment:
What is this file?

Reply:
draft file to call router contract, removed